docker-compose-db.yaml

```diff
@@ -0,0 +1,15 @@
+version: "3.7"
```

Comment:
Is there reason for this docker-compose as opposed to `docker-compose.services.yaml` which includes postgres already?